alpha/alpha-server/src/test/java/io/servicecomb/saga/alpha/server/AlphaIntegrationTest.java

```diff
@@ -80,14 +82,32 @@
   @Autowired
   private List<CompensationContext> compensationContexts;
 
+  // use an empty response observer as we don't need the response in client side
+  private final StreamObserver<GrpcCompensateCommand> emptyResponseObserver = new StreamObserver<GrpcCompensateCommand>() {
```

Comment:
do we need an integration test in PackIT to verify the compensation callback between alpha and omega?